pkg/util/afero/afero.go

```diff
@@ -27,6 +27,10 @@ func CopyDir(fs afero.Fs, src, dst string) error {
 	if err != nil {
 		return err
 	}
+	err = fs.Mkdir(dst, 0o755)
```

Comment:
MkdirAll?
There's no guarantee that caller will ensure parent of dst exists.

Reply:
I intend those semantics, when we recursively call this we will have created the parents. But for anyone calling this function I wanted it to behave like "Copy" which requires the target parent directory to already exist.